mslice/plotting/plot_window/plot_figure.py

```diff
@@ -139,6 +140,24 @@ def print_plot(self):
             painter.drawPixmap(0,0,pixmap_image)
             painter.end()
 
+    def save_plot(self):
+        file_path = QtWidgets.QFileDialog.getSaveFileName(
+            self, filter="Image (*.png);; PDF (*.pdf);; Nexus (*.nxs);; Ascii (*.txt);; Matlab (*.mat)")
+        ext = file_path[file_path.rfind('.'):]
+        save_name = os.path.basename(file_path)
+        file_path = os.path.dirname(file_path)
+        title = self._plot_handler._ws_title
```

Comment:
I think `_ws_title` should be just `title` - it gives an error otherwise for me.

Also, when I click on the save icon in a cut window, and then select a file type and just type the name of the file (without extension) it always saves a `png` whichever file type I select. This is because the `QFileDialog` is just returning the name I type without adding the file extension.

However, even when I type the filename with extension, it only saves a `png` or `pdf` - typing `nxs`, `txt`, or `mat` results in nothing being saved, a no error messages given either.